bin/main.rs

```diff
@@ -209,6 +216,20 @@ async fn main() -> anyhow::Result<()> {
         COMMAND_STREAM_NAME.to_owned(),
         vec![DEFAULT_COMMANDS_TOPIC.to_owned()],
         Some("A stream that stores all commands for wadm".to_string()),
+        RetentionPolicy::WorkQueue,
+        None,
+        None,
+    )
+    .await?;
+
+    let status_stream = nats::ensure_stream(
+        &context,
+        STATUS_STREAM_NAME.to_owned(),
+        vec![DEFAULT_STATUS_TOPIC.to_owned()],
+        Some("A stream that stores all status updates for wadm applications".to_string()),
+        RetentionPolicy::Limits,
+        Some(std::time::Duration::from_nanos(0)),
+        Some(10),
```

Comment:
Choosing here to store status forever time-wise, but only retain the last 10 statuses for manifests.

Reply:
Yep, this seems smart enough for now